custom/enikshay/integrations/bets/repeater_generators.py

```diff
@@ -220,6 +224,15 @@ def create_voucher_payload(cls, voucher_case):
             related_case_id=voucher_case.case_id
         )
 
+        approver_id = voucher_case.get_case_property('voucher_approved_by_id')
+        if approver_id:
+            approver = CommCareUser.get_by_user_id(approver_id)
+            approver_name = approver.name
+            approver_usertype = approver.user_data.get('usertype')
+        else:
```

Comment:
in theory we should only be sending them approved vouchers. If this property isn't set, then something has gone wrong somewhere...